baywatch.install

```diff
@@ -423,3 +423,14 @@ function baywatch_update_8021() {
   $baywatch = new BaywatchOperation();
   $baywatch->enable_autologout();
 }
+
+/**
+ * Uninstall admin_toolbar_search module
+ */
+function baywatch_update_8022() {
+  $module_installer = \Drupal::service('module_installer');
+  $module_handler = \Drupal::moduleHandler();
+  if ($module_handler->moduleExists('admin_toolbar_search')) {
+    $module_installer->uninstall(['admin_toolbar_search']);
+  }
+}
```

Comment:
@Jeannie-deloitte 
This is a just a small ask, but configure your editor to always include a linebreak at the end of a file. It helps avoid unnecessary conflicts and whitespace only commits.